.github/workflows/release.yml

```diff
@@ -0,0 +1,104 @@
+name: Build and upload binaries
+
+on:
+  repository_dispatch:
+    types: [tag-created]
+
+jobs:
+  release:
+    name: Build and release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            artifact_name: manta-signer
+            asset_name: manta-signer-linux-amd64
+          - os: windows-2019
+            artifact_name: manta-signer
+            asset_name: manta-signer-windows
+          - os: macos-latest
+            artifact_name: manta-signer
+            asset_name: manta-signer-macos-amd64
+    #      - os: macos-10.15
+    #        artifact_name: manta-signer
+    #        asset_name: manta-signer-macos-amd64
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Cache Cargo build dir
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-cargo
+        with:
+          path: ~/.cargo
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('./ui/src-tauri/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Cache target dir
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-target
+        with:
+          path: ui/src-tauri/target
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('./ui/src-tauri/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Install Tauri CLI
+        run: cargo install tauri-cli --git 'https://github.com/tauri-apps/tauri' --branch next
+      - name: Fetch reclaim_pk.bin
+        run: cd ui/src-tauri;curl -O 'https://github.com/Manta-Network/manta-api/releases/download/0.3.1/reclaim_pk.bin'
+      - name: Fetch transfer_pk.bin
+        run: cd ui/src-tauri;curl -O 'https://github.com/Manta-Network/manta-api/releases/download/0.3.1/transfer_pk.bin'
```

Comment:
These should be fetched from https://github.com/Manta-Network/sdk. In the mid-to-long term we want to make downloading from the SDK very easy, just like running one command or something.